README.md

```diff
@@ -100,11 +100,10 @@ New-Item -Path .git/hooks/pre-commit -ItemType HardLink -Value <absolute_path/to
 We use `mdbook-linkcheck` to validate URLs included in our documentation. To perform link checks, uncomment the `[output.linkcheck]` field in the `book.toml` configuration file and install `mdbook-linkcheck` with:
 
 ```bash
-> cargo install mdbook-linkcheck --git https://github.com/Michael-F-Bryan/mdbook-linkcheck --rev 14441d77646d58cea8ffc32fde9ea33b2bedb1a2
+> cargo install mdbook-linkcheck
 ```
 
-Note that we use an alpha version of `mdbook-linkcheck` to be able to use a feature that hasn't landed in a release yet.
-You will also need `mdbook` version `>= 0.3.5`.
+You will need `mdbook` version `>= 0.3.5` and `mdbook-linkcheck` version `>= 0.7.1` to check links.
 `linkcheck` will be run automatically when you build with the instructions in the section above.
 
 **Please note**: You may receive errors like the following when link checks are active on local `mdbook` builds:
```

Comment:
Is this still relevant now that it's not checking all links?

Reply:
I think it's still theoretically possible (e.g. if a file has a ton of links or if you touch a bunch of files). I thinks it's ok to keep this.

Reply:
Yes, I thought of removing it too, but kept it for the reasons mark mentioned.